Tests/KituraTests/TestCodableRouter.swift

```diff
@@ -264,6 +265,46 @@ class TestCodableRouter: KituraTest {
             .run()
     }
 
+    func testBasicGetIdentifiersArray() {
+        var storeTuple = [(Int, User)]()
+        self.userStore.forEach { storeTuple.append(($0.0, $0.1)) }
+        
+        router.get("/users") { (respondWith: ([(Int, User)]?, RequestError?) -> Void) in
+            print("GET on /users")
+            respondWith(storeTuple, nil)
+        }
+        
+        performServerTest(router, timeout: 30) { expectation in
+            self.performRequest("get", path: "/users", callback: { response in
+                guard let response = response else {
+                    XCTFail("ERROR!!! ClientRequest response object was nil")
+                    return
+                }
+                
+                XCTAssert(response.headers.contains { (key: String, value: [String]) in return key == "Content-Type" && value.contains("application/json") })
+                XCTAssertEqual(response.statusCode, HTTPStatusCode.OK, "HTTP Status code was \(String(describing: response.statusCode))")
+                var data = Data()
+                guard let length = try? response.readAllData(into: &data) else {
+                    XCTFail("Error reading response length!")
+                    return
+                }
+                
+                XCTAssert(length > 0, "Expected some bytes, received \(String(describing: length)) bytes.")
+                guard let users = try? JSONDecoder().decode([[Int: User]].self, from: data) else {
```

Comment:
Converting the `String` in `JSON`, to `Int` in Swift inside the JSONDecoder(), should this be the behaviour?
 Or should we return always a `String`? @Andrew-Lees11 will play around with it.

Reply:
Actually the `Identifier` protocol had a constructor which accepts a `String` which means that is fine to return a  `String` since we can always construct the `Identifier` from that `String`